common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
Some clipping caused by duplicate meshes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120005726-2e4cc380-bfe9-11eb-9ff7-1be9e1cf848e.png)

</details>